### PR TITLE
[FEAT] Add referral code to profile page

### PR DESCRIPTION
**Pull request by benjaminkostiuk**
## Overview

Adds the referral code to the profile page + minor bug fixes.

**This build requires a rebuild of the native dev client**

This change is a

- [x] Bug fix
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that will change existing functionality)
- [ ] Documentation update

## Preview
Animation

## Description

- Add expo clipboard
- Add new theme color for refresh icon background.
- Add a copy to clip function in profile. Can be moved to utils if it's needed elsewhere.
- Add referral code to profile page.
- Fix padding at top of wait times page.

## Motivation/Links
Show users their referral codes.

### CODE REVIEW

File: src/screens/WaitTimeScreen.tsx [hunk 1]
```diff
@@ -271,7 +272,7 @@ export const WaitTimeScreen = ({ navigation }: IWaitTimeScreenProps) => {
             <RefreshControl
               refreshing={refreshing}
               onRefresh={onRefresh}
-              progressBackgroundColor={theme.iconColor}
+              progressBackgroundColor={theme.refreshIconBackgroundColorAndroid} // Android only: Background color of refresh icon
```
Comment (dhruv423) [inline src/screens/WaitTimeScreen.tsx:275]: Doesn't `colours` prop work?

Comment (benjaminkostiuk) [reply src/screens/WaitTimeScreen.tsx:275]: Sorry what are you suggesting?

Comment (dhruv423) [reply src/screens/WaitTimeScreen.tsx:275]: Im not sure what this does, but this https://reactnative.dev/docs/refreshcontrol#colors-android for android

File: src/screens/WaitTimeScreen.tsx [hunk 2]
```diff
@@ -320,7 +321,7 @@ const styles = StyleSheet.create({
     paddingBottom: 0,
   },
   searchContainer: {
-    paddingTop: 35,
+    paddingTop: Platform.OS === "ios" ? 35 : 0,
```
Comment (dhruv423) [inline src/screens/WaitTimeScreen.tsx:324]: Gonna test this, waiting for build to finish ugh

Comment (benjaminkostiuk) [reply src/screens/WaitTimeScreen.tsx:324]: Fucking clipboard 😡

File: src/screens/WaitTimeScreen.tsx [hunk 3]
```diff
@@ -360,7 +361,7 @@ const styles = StyleSheet.create({
     borderTopRightRadius: 10,
     paddingBottom: 10,
     paddingHorizontal: 20,
-    paddingTop: 20,
+    paddingTop: 5,
```
Comment (dhruv423) [inline src/screens/WaitTimeScreen.tsx:364]: I think 5 is too small, 10 seems like a good middle

Comment (benjaminkostiuk) [reply src/screens/WaitTimeScreen.tsx:364]: Feel free to push the change tonight, or I can do it tmmrw.

**Review by dhruv423 — APPROVED** [review]

**Pull request merged**